src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -558,10 +557,9 @@ def load_command_table(self, _):
                             client_factory=cf_shared_gallery_image_version) as g:
         g.custom_command('list-shared', 'sig_shared_image_version_list')
 
-    with self.command_group('sig gallery-application', compute_gallery_application_sdk, client_factory=cf_gallery_application, min_api='2021-07-01', operation_group='gallery_applications') as g:
-        g.custom_command('create', 'gallery_application_create', supports_no_wait=True)
-        g.custom_command('update', 'gallery_application_update', supports_no_wait=True)
-        g.wait_command('wait')
+    with self.command_group('sig gallery-application', compute_gallery_application_profile, min_api='2021-07-01', operation_group='gallery_applications') as g:
```

Comment:
The `min_api` may can be removed since the `commands.py` file directly under `vm` folder is only for the latest profile
```suggestion
    with self.command_group('sig gallery-application', compute_gallery_application_profile, operation_group='gallery_applications') as g:
```